### PR TITLE
Editor atlas support

### PR DESCRIPTION
**Pull request by Jhonnyg**
Requires Defold 1.4.5+

### CODE REVIEW

File: defold-rive/editor/src/rive.clj [hunk 1]
```diff
@@ -506,10 +511,13 @@
       (restore-gl gl))))
 
 (defn- render-rive-scenes [^GL2 gl render-args renderables rcount]
-  (let [pass (:pass render-args)]
+  (let [pass (:pass render-args)
+        render-groups (collect-render-groups renderables)]
+       (doseq [group render-groups]
+         (plugin-update-file (:handle group) 0.0 (:texture-set-pb group)))
```
Comment (matgis) [inline defold-rive/editor/src/rive.clj:517]: This seems very expensive to do on every render call. Shouldn't we try to cache it as an output on the `RiveModelNode`?

File: defold-rive/pluginsrc/rive_file.cpp [hunk 2]
```diff
@@ -298,14 +301,55 @@ static void Render(RiveFile* rive_file)
...
+            case DRAW_MODE_DEFAULT:
+                dmLogInfo("  DRAW_MODE_DEFAULT");
+                break;
+            case DRAW_MODE_CLIP_DECR:
+                dmLogInfo("  DRAW_MODE_CLIP_DECR");
+                break;
+            case DRAW_MODE_CLIP_INCR:
+                dmLogInfo("  DRAW_MODE_CLIP_INCR");
+                break;
+        }
```
Comment (matgis) [inline defold-rive/pluginsrc/rive_file.cpp:315]: Do we really want to have this log spam in the render function?

**Review by matgis — APPROVED** [review]
I think we could perhaps do a better job of caching in the editor code, but we could address that later. Looks good to merge.

File: defold-rive/editor/src/rive.clj (hunk 1 above)
Comment (Jhonnyg) [reply defold-rive/editor/src/rive.clj:517]: Yeah, sounds good 👍

File: defold-rive/pluginsrc/rive_file.cpp (hunk 2 above)
Comment (Jhonnyg) [reply defold-rive/pluginsrc/rive_file.cpp:315]: Eeeh oops, no!

**Pull request merged**